aws/resource_aws_ssm_association.go

```diff
@@ -75,7 +75,7 @@ func resourceAwsSsmAssociation() *schema.Resource {
 				Optional: true,
 				ForceNew: true,
```

Comment:
Since we're now able to update this I think we should also remove `ForceNew` from here, otherwise the new code will never be executed. 😉